content/developer/reference/frontend/services.rst

```diff
@@ -131,6 +133,48 @@ Reference List
    * - :ref:`user <frontend/services/user>`
      - provides some information related to the current user
 
+.. _frontend/services/company:
+
+Company service
+---------------
+
+Overview
+~~~~~~~~
+
+* Technical name: `company`
+* Dependencies: `user`, `router`, `cookie`
+
+Odoo has a concept of companies. You can manage several with the same Odoo instance. 
+More than one companies can be *active* at any time, but one company is always considered selected.
+The selected company is basically considered the current company, but for pages that support it,
+the content of all the active companies are displayed. By example, in the product page, the company A may be the main one
+but the products of all the active companies are shown. 
```

Comment:
in the backend, I don't think there's a notioni of "pages that support multi company". It's the way all models work, isn't it?